src/main/java/org/radarbase/management/service/dto/ProjectDTO.java

```diff
@@ -37,6 +37,8 @@
 
     private String organization;
 
+    private String organizationName;
+
```

Comment:
I'd expect an `OrganisationDTO` object here as well.

Reply:
I decided not to add it — we still have the old `organization` field. It would be great to remove it first once we are certain that nobody uses it — and re-use the field name. However, even this improvement is likely unnecessary, as any client that needs organization data can fetch it.